guides/getting-started.md

```diff
@@ -61,15 +61,16 @@ import 'hammerjs';
 ```
 
 ## Configuring SystemJS
-If your project is using SystemJS for module loading, you will need to add `@angular/material` 
+If your project is using SystemJS for module loading, you will need to add `@angular/material` and `hammerjs`
 to the SystemJS configuration:
 
 ```js
 System.config({
   // existing configuration options
   map: {
     ...,
-    '@angular/material': 'npm:@angular/material/bundles/material.umd.js'
+    '@angular/material': 'npm:@angular/material/bundles/material.umd.js',
+    'hammerjs': 'npm:hammerjs/hammer.js'
```

Comment:
Yeah, and also since HammerJS is kind of optional now, we should mention that it's not required.